src/resources/assets/js/components/dismissImageGridImage.vue

```diff
@@ -41,6 +64,32 @@ export default {
         title() {
             return this.selected ? 'Undo dismissing this annotation' : 'Dismiss this annotation';
         },
+        isPoint() {
+            return this.svg !== null ? this.svg.includes('r="3"') : false;
```

Comment:
Maybe add an explicit attribute to the point element in the SVG and check this here instead? The current approach is dangerous because if the radius is changed in the backend, it no longer works correctly in the frontend for no obvious reason.